packages/website/docs/_samples/main/Input/CustomIcon/CustomIcon.md

```diff
@@ -0,0 +1,11 @@
+```html title="example.html"
+<ui5-input value="Some initial value" show-clear-icon >
+    <ui5-icon id="custom-icon" name="search" slot="icon"></ui5-icon>
+</ui5-input>
+<script type="module">
+    import inputCss from "@ui5/webcomponents/dist/styles/InputIcon.module.css";
```

Comment:
IMO should be `Input.module.css` (but for now it only exposes custom icon styling, while in the future it may expose some more input styling classes for different situations)

Because currently you have `inputCss` from `InputIcon.module.css`, so it should be called `inputIconCss` to be precise. But then, it would have a property `input-icon`, which is redundant